example/macos/Podfile

```diff
@@ -7,3 +7,9 @@ use_test_app! do |target|
     pod 'Example-Tests', :path => '..'
   end
 end
+
```

Comment:
Due to the "include" above adding the target, the pod add RNSentry was added there instead ('../node_modules/react-native-test-app/macos/test_app.rb') hence doesn't show up in this PR